temporal/api/cloud/identity/v1/message.proto

```diff
@@ -70,12 +70,24 @@ message User {
     google.protobuf.Timestamp last_modified_time = 8;
 }
 
+message GoogleGroupSpec {
+    // The email address of the Google group.
+    // The email address is immutable. Once set during creation, it cannot be changed.
+    string email_address = 1;
+}
+
 message UserGroupSpec {
-    // The name of the group as defined in the customer's IdP (e.g. Google group name in Google Workspace)
-    // The name is immutable. Once set, it cannot be changed
-    string name = 1;
-    // The access assigned to the group
+    // The display name of the group.
+    string display_name = 1;
+    // The access assigned to the group.
     Access access = 2;
+    // The type of the group.
+    // Currently, the only supported type is 'google-group'.
+    // The type is immutable. Once set during creation, it cannot be changed.
+    string type = 3;
```

Comment:
we can chat about this today. last time @mattkim suggested not exposing this field yet. maybe the presence of GoogleGroupSpec would indicate the group type implicitly? I will take a look at the implementation to see the necessity of exposing this field now. but just wanna leave a reminder for myself to discuss